robot/Cumulus/resources/locators_49.py

```diff
@@ -224,6 +225,7 @@
         'gift-amount':'//div[./label[text()="{}"]]',
         'modal-link':'//tbody/tr/td/a[text()="{}"]',
         'datepicker_open':"//div[contains(@class,'slds-is-open')][./label[contains(text(),'{}')]]",
+        'lookup_option': "//span/lightning-base-combobox-formatted-text[@title='{}']"
```

Comment:
the locator  under ['gift_entry']['lookup-option'] should work as both pages use the standard lookup. see if u could you the same locator instead of creating a new one again